### DISCUSSION TITLE
Is it possible to change the design (layout) of the log using themes?

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2026] [Ideas / Feature Requests]
Is it possible to change the design (layout) of the log using themes?
The map now takes up a disproportionate amount of space that could be used to display logbook entries.
Is it possible to change the layout of the objects on the log page using themes as suggested in the image? Or do I have to change the code (php)?

![изображение](https://user-images.githubusercontent.com/16667173/223137982-07049363-4d38-4024-9bda-042cb2ee08fa.png)

**Comment by magicbug** [comment]
Todo this you'd need change the core code.

**Reply by R1BLH** [reply]
I see.
Unfortunately, I'm not qualified enough in web programming to do this easily.
I tried to change the location in Firefox, but the result was that the QSO markers disappeared from the map :)

Otherwise the map takes up a lot of space and is often not very informative, especially on the logbook overview page. It would be nice to be able to hide it from this page in the settings.

Another question: how can I add more tags for localisation? 

![изображение](https://user-images.githubusercontent.com/16667173/223142980-e0243c67-fcb5-40b6-af66-901ef2819b94.png)

**Reply by R1BLH** [reply]
> Todo this you'd need change the core code.

I did this, i hope. :)
Could you check PR #2028, please?

**Reply by R1BLH** [reply]
> Todo this you'd need change the core code.

Could you advice how to make default map leaflet position and scale like on the screenshot, please?
![изображение](https://user-images.githubusercontent.com/16667173/223407012-b643d92b-06db-46b4-bb8a-c24c2eadd09d.png)
